SQL/paradise_schema.sql

```diff
@@ -250,6 +250,7 @@ CREATE TABLE `player` (
   `nanoui_fancy` smallint(4) DEFAULT '1',
   `show_ghostitem_attack` smallint(4) DEFAULT '1',
   `lastchangelog` varchar(32) NOT NULL DEFAULT '0',
+  `exp` mediumtext NOT NULL DEFAULT '',
```

Comment:
The `mediumtext` type cannot have a default value ([relevant page in MySQL reference manual](http://dev.mysql.com/doc/refman/5.7/en/blob.html)). You may want to use a varchar instead.